src/test/resources/po_listed_print_monograph_pe_mix.json

```diff
@@ -0,0 +1,148 @@
+    "ABCDEFGHIJKLMNOPQRST",
+    "ABCDEFGHIJKLMNOPQRSTUV"
+  ],
+  "order_type": "One-Time",
+  "po_number": "268758",
+  "re_encumber": false,
+  "total_estimated_price": 49.98,
+  "total_items": 2,
+  "vendor": "168f8a86-d26c-406e-813f-c7527f241ac3",
+  "workflow_status": "Pending",
+  "metadata": {
+    "createdDate": "2010-10-08T03:53:00.000",
+    "createdByUserId": "ab18897b-0e40-4f31-896b-9c9adc979a88"
+  },
+  "compositePoLines": [
+    {
+      "id": "c0d08448-347b-418a-8c2f-5fb50248d67e",
+      "acquisition_method": "Purchase",
+      "adjustment": {
+        "credit": 0.0,
+        "discount": 0.0,
+        "insurance": 0.0,
+        "invoice_id": "2ddfeb5c-1237-476f-aa88-57f7cbf74ca4",
+        "overhead": 0.0,
+        "shipment": 0.0,
+        "tax_1": 0.0,
+        "tax_2": 0.0,
+        "use_pro_rate": false,
+        "po_line_id": "c0d08448-347b-418a-8c2f-5fb50248d67e"
+      },
+      "alerts": [],
+      "cancellation_restriction": false,
+      "cancellation_restriction_note": "",
+      "claims": [
+        {
+          "claimed": false,
+          "grace": 0,
+          "po_line_id": "c0d08448-347b-418a-8c2f-5fb50248d67e"
+        }
+      ],
+      "collection": false,
+      "contributors": [
+        {
+          "contributor": "Ed Mashburn",
+          "contributor_type": "fbdd42a8-e47d-4694-b448-cc571d1b44c3"
+        }
+      ],
+      "cost": {
+        "list_price": 25.10,
+        "currency": "USD",
+        "quantity_physical": 3,
+        "quantity_electronic": 2,
+        "po_line_estimated_price": 75.30,
+        "po_line_id": "c0d08448-347b-418a-8c2f-5fb50248d67e"
+      },
+      "description": "",
+      "details": {
+        "receiving_note": "",
+        "product_ids": [
+          {
+            "product_id": "9781118416068",
+            "product_id_type": "ISBN"
+          }
+        ],
+        "material_types": [
+          "615b8413-82d5-4203-aa6e-e37984cb5ac3"
+        ],
+        "subscription_from": null,
+        "subscription_interval": 0,
+        "subscription_to": null,
+        "po_line_id": "c0d08448-347b-418a-8c2f-5fb50248d67e"
+      },
+      "donor": "",
+      "eresource": {
+        "create_inventory": false
+      },
+      "fund_distribution": [
+        {
+          "code": "MISCHIST-FY19",
+          "percentage": 100.0,
+          "encumbrance": "eb506834-6c70-4239-8d1a-6414a5b08015",
+          "po_line_id": "c0d08448-347b-418a-8c2f-5fb50248d67e"
+        }
+      ],
+      "locations": [
+        {
+          "po_line_id": "c0d08448-347b-418a-8c2f-5fb50248d67e",
+          "location_id": "b241764c-1466-4e1d-a028-1a3684a5da87",
+          "quantity": 2,
+          "quantity_electronic": 2,
+          "quantity_physical": 0
+        },
+        {
+          "po_line_id": "c0d08448-347b-418a-8c2f-5fb50248d67e",
+          "location_id": "758258bc-ecc1-41b8-abca-f7b610822ffd",
+          "quantity": 0,
+          "quantity_electronic": 0,
+          "quantity_physical": 0
```

Comment:
Is it intended to be 0? According to `cost` content, 3 physical and 2 electronic resources are expected. The first location covers electronic resources. I believe the second one is intended for physical but quantity is 0 which means create nothing in Inventory for this location.

Reply:
Got it. I have corrected it and made corresponding changes to the implementation.